### PR TITLE
[CORE-1233] Add kubernetes event logs to pachctl

### PR DESCRIPTION
**Pull request by armaanv**
This PR adds a new pachctl command `pachctl kube-events` which returns the logs from the kube-event-tail pod that is deployed along with pachyderm

### CODE REVIEW

**Comment by armaanv** [comment]
I updated this to go through Loki. The RPC now takes in a LokiRequest message and outputs LokiLogMessages. These can be used for future additions to the API. It's also a streaming RPC now. I've also added in some formatting for the output along with a couple of flags.

File: src/server/pps/cmds/cmds.go [hunk 1]
```diff
@@ -539,6 +539,42 @@ each datum.`,
...
+		Long:  "Return the kubernetes events.",
+		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			client, err := pachdclient.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			since, err := time.ParseDuration(since)
+			if err != nil {
+				return errors.Wrapf(err, "error parsing since(%q)", since)
```
Comment (jrockway) [inline src/server/pps/cmds/cmds.go:555]: Errors need not contain the word "error".  `parse since(%q)` is fine.

**Review by jrockway — APPROVED** [review]
Looks good.  Does the tablewriter thing treat \t specially?  I feel like there isn't enough space in 8 characters to print any of that stuff.

File: src/server/pps/cmds/cmds.go (hunk 1 above)
Comment (armaanv) [reply src/server/pps/cmds/cmds.go:555]: will change this

**Comment by armaanv** [comment]
> Looks good. Does the tablewriter thing treat \t specially? I feel like there isn't enough space in 8 characters to print any of that stuff.

Yeah, it uses tabs to delimit the columns in the output

**Pull request merged**